.travis.yml

```diff
@@ -46,17 +46,27 @@ jobs:
       before_cache:
 
     - stage: Linter + Doxygen + non-debug Ubuntu/gcc-5 test
-      env: NAME="DOXYGEN-CHECK"
+      env:
+        NAME: "DOXYGEN-CHECK"
+        DOXYGEN_VERSION: "1.8.14"
       addons:
         apt:
           sources:
             - sourceline: 'deb http://packages.cloud.google.com/apt cloud-sdk-trusty main'
               key_url: 'https://packages.cloud.google.com/apt/doc/apt-key.gpg'
           packages:
-            - doxygen
+            - cmake
             - google-cloud-sdk
       install:
-      script: scripts/travis_doxygen.sh
+        - wget http://ftp.stack.nl/pub/users/dimitri/doxygen-${DOXYGEN_VERSION}.src.tar.gz
+        - tar xfz doxygen-${DOXYGEN_VERSION}.src.tar.gz
+        - mkdir -p doxygen-${DOXYGEN_VERSION}/build
+        - cd doxygen-${DOXYGEN_VERSION}/build; cmake ..; make -j4
+        - export PATH=$PATH:`pwd`/bin
```

Comment:
`export PATH="$PATH:${TRAVIS_BUILD_DIR}/doxygen-${DOXYGEN_VERSION}/bin"` will be more reliable.